microsite/data/plugins/jira-dashboard

```diff
@@ -0,0 +1,10 @@
+---
+title: Jira Dashboard
+author: AxisCommunications
+authorUrl: https://github.com/AxisCommunications
+category: Agile Planning
+description: Allows you to fetch and display Jira issues for your entity. You get quickly access to issue summaries to achieve better task visibility and more efficient project management. 
+documentation: https://github.com/AxisCommunications/backstage-plugins
```

Comment:
You'll want to link them to the specific documentation for this plugin, I would use this: https://github.com/AxisCommunications/backstage-plugins/blob/main/plugins/jira-dashboard/README.md